Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM alpine:3.3
 MAINTAINER Nicolas Degory <ndegory@axway.com>
 
+#Need to keep curl and bash after installation for containerPilot
 RUN apk update && \
-    apk --no-cache add python ca-certificates && \
-    apk --virtual envtpl-deps add --update py-pip python-dev curl && \
+    apk --no-cache add python ca-certificates curl bash && \
```

Comment:
Curl is going to be used to check the availability of InfluxDB on a better way than the current "pidof influxd". We can remove Curl and re-add it later when needed.